src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ CHANGELOG
    `RoutingResolverPass`, `SerializerPass`, `ValidateWorkflowsPass`
  * made  `Translator::__construct()` `$defaultLocale` argument required
  * removed `SessionListener`, `TestSessionListener`
+ * Removed `cache:clear` warmup part along with the `--no-optional-warmers` option
+ * Removed core form types services registration when unnecessary
+ * Removed `framework.serializer.cache` option and `serializer.mapping.cache.apc`, `serializer.mapping.cache.doctrine.apc` services
+ * Removed `ConstraintValidatorFactory::$validators` and `ConstraintValidatorFactory::$container` protected properties
+ * Removed class parameters related to routing
```

Comment:
Sure, it's mentioned in https://github.com/symfony/symfony/blob/master/UPGRADE-3.3.md#frameworkbundle:

> Class parameters related to routing have been deprecated and will be removed in 4.0.